src/external/sw/index.ts

```diff
@@ -0,0 +1,77 @@
+/// <reference no-default-lib="true"/>
+/// <reference lib="esnext" />
+/// <reference lib="WebWorker" />
```

Comment:
Annotation? why write like this?

Reply:
Because service-worker requires specific lib in TypeScript. I don't want to add a new tsconfig here, so type annotation is a good choice.